Source/WebService/Blackjack.WebService/ClientApp/src/main.ts

```diff
@@ -8,13 +8,16 @@ export function getBaseUrl() {
   return document.getElementsByTagName('base')[0].href;
 }
 
-const providers = [
-  { provide: 'BASE_URL', useFactory: getBaseUrl, deps: [] }
-];
+//const providers = [
```

Comment:
Now that we understand what this shenanigans is from our investigation of the proxy setup we can safely remove it from this file.

We won't ever need to uncomment it.